plugins/template.md

```diff
@@ -173,35 +209,81 @@ This can then be used in a template or snippet that looks like this:
 
 ```html
 <p>Name: {$username}, StaffID: {$staffid}</p>
-``` 
+```
 
-And that will be changed to: 
+And that will be changed to:
 
 ```html
 <p>Name: Jack Black, StaffID: 991234</p>
 ```
 
-### `template_selected_content_classes`
+### `template_preview_replace_values`
 
-This option allows you to provide a list of class names (separated by spaces) whereby any template element with one of the classes will have its content replaced by the selected editor content when first inserted.
+This is an object containing items that will be replaced with their respective string values in the template preview shown in the template dialog.
 
+**Type:** `Object`
 
 ##### Example
 
 ```js
 tinymce.init({
   selector: "textarea",  // change this value according to your HTML
   plugins: "template",
-  template_selected_content_classes: "selcontent selectedcontent"
+  menubar: "insert",
+  toolbar: "template",
+  template_preview_replace_values: {
+    username: "Jack Black",
+    staffid: "991234"
+  }
 });
 ```
 
-### Template Plugin Examples
+This can then be used in a template or snippet that looks like this:
+
+```html
+<p>Name: {$username}, StaffID: {$staffid}</p>
+```
+
+And the preview will look like:
+
+```html
+<p>Name: Jack Black, StaffID: 991234</p>
+```
+
+### `template_selected_content_classes`
+
+When HTML elements in a template are assigned this content class, the content of the element will be replaced with selected content from the editor. This option accepts a list of content classes (separated by spaces).
 
 **Type:** `String`
 
+**Default Value:** `selcontent`
+
 ##### Example
 
+```js
+tinymce.init({
+  selector: "textarea",  // change this value according to your HTML
+  plugins: "template",
+  template_selected_content_classes: "selcontent selectedcontent",
+  templates: [
+    {
+      title: 'My Template',
+      description: 'This is my template.',
+      content: '<p>Hello, <span class="selcontent">this statement will be replaced.</span></p>'
+    }
+  ]
+});
+```
+If the word `world` is selected in the editor and _My Template_ is applied, `world` will be updated to:
```

Comment:
I was a little confused at first because there's no context for what the editor content is. But could also be because I made that fiddle, and this is similar enough that that's where my brain went first. Should probably get someone else to read it to check?

Reply:
Fair enough.
I ran it by @thedrjim, who thinks it checks out.